app/app/management/commands/sync_orgs_repos.py

```diff
@@ -0,0 +1,113 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User, Group
+from dashboard.models import Profile, Organization
+from git.utils import get_organization, get_user, get_repo
+from app.utils import sync_profile
+
+
+class Command(BaseCommand):
+    help = 'Synchronizes Organizations and Repo roles to members'
+
+    # def add_arguments(self, parser):
+
+    def handle(self, *args, **options):
+
+        try:
+            print("Loading Users....")
+            all_users = User.objects.all()  # potentially want to add some filters here around what users are synced
+            print(all_users)
+            print("Looking up Organization of user")
+            # memoize the process so we only ever sync once per user
+
+            synced = []
+
+            def recursive_sync(lsynced, handle):
+                try:
+                    if handle not in lsynced:
+                        print(f'Syncing User Handle: {handle}')
+                        profile = sync_profile(handle)
+                        remove = [x for x in profile.orgs.all() if x.name not in []]
```

Comment:
Whats happening here? isn't this the same as `remove = profile.orgs.all()` since `x.name not in []` is always true?

Reply:
Looks like I forgot to put my conditional back in when i was testing, should be checking to ensure that the old orgs are actually not in the profile.organizations that is synced from the profile check.

Reply:
This is how I do reconciliation per user.